docs/source/starter/introduction.rst

```diff
@@ -12,103 +12,58 @@
-Now you can train your models on different accelerators like GPU/TPU/IPU, to do distributed training across multiple machines/nodes without code changes using state-of-the-art distributed training mechanisms.
+Organizing your code with PyTorch Lightning makes your code:
 
-Code organization is the core of Lightning. It leaves the research logic to you and automates the rest.
-
-----------
-
-********************
-Lightning Philosophy
-********************
-
-Organizing your code with Lightning makes your code:
-
-* Flexible (this is all pure PyTorch), but removes a ton of boilerplate
+* Keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
 * More readable by decoupling the research code from the engineering
 * Easier to reproduce
 * Less error-prone by automating most of the training loop and tricky engineering
 * Scalable to any hardware without changing your model
 
-Lightning is built for:
-
-* Researchers who want to focus on research without worrying about the engineering aspects of it
-* ML Engineers who want to build reproducible pipelines
-* Data Scientists who want to try out different models for their tasks and build-in ML techniques
-* Educators who seek to study and teach Deep Learning with PyTorch
-
-The team makes sure that all the latest techniques are already integrated and well maintained.
-
 
 ----------
 
+Here's a 3 minute conversion guide for PyTorch projects:
 
-*****************
-Starter Templates
-*****************
+.. raw:: html
 
-Before installing anything, use the following templates to try it out live:
+    <video width="100%" max-width="800px" controls autoplay muted playsinline
+    src="https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/pl_docs/pl_docs_animation_final.m4v"></video>
 
-.. list-table::
-   :widths: 18 15 25
-   :header-rows: 1
+----------
 
-   * - Use case
-     - Description
-     - link
-   * - Scratch model
-     - To prototype quickly / debug with random data
-     -
-        .. raw:: html
+*********************************
+Step 0: Install PyTorch Lightning
+*********************************
 
-            <div style='width:150px;height:auto'>
-                <a href="https://colab.research.google.com/drive/1rHBxrtopwtF8iLpmC_e7yl3TeDGrseJL?usp=sharing>">
-                    <img alt="open in colab" src="http://bit.ly/pl_colab">
-                </a>
-            </div>
-   * - Scratch model with manual optimization
-     - To prototype quickly / debug with random data
-     -
-        .. raw:: html
 
-            <div style='width:150px;height:auto'>
-                <a href="https://colab.research.google.com/drive/1nGtvBFirIvtNQdppe2xBes6aJnZMjvl8?usp=sharing">
-                    <img alt="open in colab" src="http://bit.ly/pl_colab">
-                </a>
-            </div>
+You can install using `pip <https://pypi.org/project/pytorch-lightning/>`_
 
+.. code-block:: bash
 
-----------
+    pip install pytorch-lightning
```

Comment:
should we link the installation page instead?

Reply:
No, we wouldn't want the users to jump off the page.